DZ22_CLOSURES/index.js

```diff
@@ -0,0 +1,11 @@
+function cacheCounter(a){
```

Comment:
In such a function you can use mixed `function` with an arrow function, your code can be shorter

```
function createSum(a) {
  let acc = a;

  return (b) => acc += b;
}

const sum = createSum(0);
```

Reply:
Usually naming with `create` prefix and returned function name

```
const sum = createSum();
const counter = createCounter();
const myObject = createMyObject();
```